### PR TITLE
New package: quiterss-0.19.4

### PR DESCRIPTION
**Pull request by toluschr**
#23272
I'm not sure about [Line 15](https://github.com/toluschr/void-packages/blob/d6997c9001b614783676388edf32cce90fc6861f/srcpkgs/quiterss/template#L15)

### CODE REVIEW

**Comment by fosslinux** [comment]
Line 15 is incorrect. Have a look at this: https://github.com/void-linux/void-packages/blob/master/Manual.md#about-the-many-types-of-depends-variable. Ie, these two packages should be in makedepends.

**Comment by toluschr** [comment]
> Line 15 is incorrect. Have a look at this: https://github.com/void-linux/void-packages/blob/master/Manual.md#about-the-many-types-of-depends-variable. Ie, these two packages should be in makedepends.

I took it from [the deepin-music template](https://github.com/void-linux/void-packages/blob/master/srcpkgs/deepin-music/template#L18-L20), since I couldn't get it to cross-compile without it.
What would be the correct way to do this?

**Comment by fosslinux** [comment]
Oh, uh, yeah, i don't get it then, not sure why the deepin-music template has that.

**Comment by toluschr** [comment]
Also, what should I do about the failed ci build?\
It's not the PR's fault that it failed. The docker dowload exceeded the maximum job length.

**Comment by ericonr** [comment]
@fosslinux  Probably some detection stuff, if I were to guess.

@toluschr You can try to build for the missing arch on your device and report it here. Or you can rebase your branch and force push it, that way CI gets another chance to run tests.

**Comment by toluschr** [comment]
@ericonr I'd rather let the CI do it again. Seems more reliable for the maintainers.

File: srcpkgs/quiterss/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+makedepends="sqlite-devel qt5-webkit-devel qt5-multimedia-devel"
+short_desc="Open-source cross-platform RSS/Atom news feeds reader"
+maintainer="toluschr <toluschr@protonmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://quiterss.org/"
+distfiles="https://github.com/QuiteRSS/${pkgname}/archive/${version}.tar.gz"
+checksum=d9dffa205a8ec4e7bf00d87183fd94d4a12f045fae04a6efd41d6557827233bc
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-webkit-devel qt5-multimedia-devel"
```
Comment (sgn) [inline srcpkgs/quiterss/template:16]: In my _very opinionated and probably incorrect_ opinions, I'll put those things into `hostmakedepends` regardless of `$CROSS_BUILD`

Comment (toluschr) [reply srcpkgs/quiterss/template:16]: Wouldn't that make building non-cross slower, because of the additional dependencies?

Comment (sgn) [reply srcpkgs/quiterss/template:16]: They'll be pulled by `makedepends` regardless.
So: no, no kittens will be harmed.

**Comment by Chocimier** [comment]
I prefer https://github.com/void-linux/void-packages/blob/6d445f0727ee5bdffa9b29646cc06a51027b22ef/srcpkgs/signond/template#L21 to headers in hosmakedepends where it work, but it don't here.

**Pull request merged**